operator/params.yaml

```diff
@@ -1,24 +1,43 @@
+################################################################################
+############################### Operator settings ##############################
+################################################################################
+
 NODE_COUNT:
-  description: "Number of Cassandra nodes"
-  displayName: "Node Count"
+  description: "Number of Cassandra nodes."
   default: 3
 
-NODE_CPUS:
-  description: "CPUs (request) for the Cassandra node pods. spec.containers[].resources.requests.cpu"
-  default: "2000m"
+NODE_CPU_MC:
+  description: "CPU request (in millicores) for the Cassandra node pods."
+  default: 2000
 
-NODE_CPUS_LIMIT:
-  description: "CPUs (limit) for the Cassandra node pods. spec.containers[].resources.limits.cpu"
-  default: "2000m"
+NODE_CPU_LIMIT_MC:
+  description: "CPU limit (in millicores) for the Cassandra node pods."
+  default: 2000
 
-NODE_MEM:
-  description: "Memory (request) for the Cassandra node pods. spec.containers[].resources.requests.memory"
+NODE_MEM_MIB:
+  description: "Memory request (in MiB) for the Cassandra node pods."
   default: 4096
 
-NODE_MEM_LIMIT:
-  description: "Memory (limit) for the Cassandra node pods. spec.containers[].resources.limits.memory"
+NODE_MEM_LIMIT_MIB:
+  description: "Memory limit (in MiB) for the Cassandra node pods."
   default: 4096
 
+NODE_DISK_SIZE_GIB:
+  description: "Disk size (in GiB) for the Cassandra node pods."
+  default: 20
+
+NODE_STORAGE_CLASS:
+  description: "The storage class to be used in volumeClaimTemplates. By default its not required and the default storage class is used."
+  required: false
+
+OVERRIDE_CLUSTER_NAME:
+  description: "Override the name of the Cassandra cluster set by the operator. This shouldn't be explicit set, unless you know what you're doing."
```

Comment:
Ah, got it. I decided to prepend an `OVERRIDE_` (also thought about `UNSAFE_`) as a way to make it sound more dangerous, since it's something that is configurable but shouldn't really be configured in normal cases. I can revert to `CLUSTER_NAME` if you think it's best.